app.rb

```diff
@@ -0,0 +1,18 @@
+require 'rubygems'
+require 'bundler'
+
+Bundler.require
+
+require './app/base'

```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.